riscv_private.h

```diff
@@ -4,6 +4,7 @@
 #include "riscv.h"
 
 #define RV_NUM_REGS 32
+#define RV_NAN 0x7fc00000
```

Comment:
`RV_NAN` should not appear in the region. Move it to somewhere after `FMASK_FRAC`.

Reply:
Should I put it here?
```
#ifdef ENABLE_RV32F
enum {
    //             ....xxxx....xxxx....xxxx....xxxx
    FMASK_SIGN = 0b10000000000000000000000000000000,
    FMASK_EXPN = 0b01111111100000000000000000000000,
    FMASK_FRAC = 0b00000000011111111111111111111111,
    //             ....xxxx....xxxx....xxxx....xxxx
    RV_NAN     = 0b01111111110000000000000000000000
};
#endif
```

Reply:
It looks fine.